Signal/src/ViewControllers/ConversationView/Cells/OWSMessageCell.m

```diff
@@ -1435,70 +483,74 @@ - (void)handleTextTapGesture:(UITapGestureRecognizer *)sender
         }
     }
 
-    if (self.hasTapForMore) {
-        [self.delegate didTapTruncatedTextMessage:self.viewItem];
-        return;
+    CGPoint locationInMessageBubble = [sender locationInView:self.messageBubbleView];
```

Comment:
Thanks to `gestureLocationForLocation` let us significantly streamline this logic.  I think it's more readable too.

Reply:
👍